packages/paste-core/components/callout/src/CalloutText.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface CalloutTextProps extends Partial<Omit<HTMLParagraphElement, 'children'>> {
+  children?: React.ReactNode;
+  element?: BoxProps['element'];
+}
+
+export const CalloutText = React.forwardRef<HTMLParagraphElement, CalloutTextProps>(
+  ({element = 'CALLOUT_TEXT', children, ...props}, ref) => (
+    <Text
+      {...safelySpreadTextProps(props)}
+      as="p"
+      ref={ref}
+      element={element}
+      color="currentColor"
```

Comment:
question: I'm curious about color being set to `currentColor` as opposed to `inherit`. Don't think it needs to change, and it might just be a code-style thing, so just was wonderin'

Reply:
I also just copied this from the old component 😅 I looked into it tho and it turns out that `inherit` takes the color from the parent component, but `currentColor` takes the color from whatever the closest thing in the DOM that has the color set is (even if its not the direct parent).